client/src/components/ui/progress.tsx

```diff
@@ -0,0 +1,85 @@
+import Image from "next/image";
+import * as React from "react";
```

Comment:
`React.useState` and `React.useEffect` can be imported as named imports for cleaner syntax:

`import { useEffect, useState } from "react";`